dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -489,7 +489,7 @@ def generate_facilitator_averages(summary)
       QUESTIONS_FOR_FACILITATOR_AVERAGES.each do |category, questions|
         facilitator_averages[facilitator][category.to_s.downcase.to_sym] = {}
         [:this_workshop, :all_my_workshops].each do |column|
-          average = (facilitator_averages[facilitator].slice(*(questions.map {|question| question[:primary_id]})).values.map {|x| x[column]}.inject(0) {|sum, n| sum + (n || 0)} || 0) / questions.size.to_f
+          average = (facilitator_averages[facilitator].slice(*(questions.map {|question| question[:primary_id]})).values.map {|x| x[column]}.reject(&:nil?).reduce(:+) || 0) / questions.size.to_f
```

Comment:
Nit: this file and this function has several lines longer than 200 characters; they are difficult to read and validate. I don't think we have to change all of them now, but we could make the lines we touch shorter or break them down into smaller pieces. I think 100-120 chars/line is acceptable.